src/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/ArtworkDetails.tsx

```diff
@@ -61,34 +73,45 @@ export const ArtworkDetails: React.FC<ArtworkDetailsProps> = ({
       : undefined
 
     if (relayEnvironment) {
+      let submissionData: SubmissionInput
+      submissionData = {
+        externalId: submission?.externalId,
+        artistID: artworkDetailsForm.artistId,
+        year: artworkDetailsForm.year,
+        title: artworkDetailsForm.title,
+        medium: artworkDetailsForm.materials,
+        attributionClass: artworkDetailsForm.rarity
+          .replace(" ", "_")
+          .toUpperCase() as ConsignmentAttributionClass,
+        editionNumber: artworkDetailsForm.editionNumber,
+        editionSizeFormatted: artworkDetailsForm.editionSize,
+        height: artworkDetailsForm.height,
+        width: artworkDetailsForm.width,
+        depth: artworkDetailsForm.depth,
+        dimensionsMetric: artworkDetailsForm.units,
+        provenance: artworkDetailsForm.provenance,
+        locationCity: artworkDetailsForm.location.city.trim(),
+        locationCountry: artworkDetailsForm.location.country?.trim(),
+        locationState: artworkDetailsForm.location.state?.trim(),
+        locationCountryCode: artworkDetailsForm.location.countryCode?.trim(),
+        locationPostalCode: artworkDetailsForm.postalCode?.trim() || null,
+        state: "DRAFT",
+        utmMedium: utmParams?.utmMedium,
+        utmSource: utmParams?.utmSource,
+        utmTerm: utmParams?.utmTerm,
+        sessionID: !isLoggedIn ? getENV("SESSION_ID") : undefined,
+      }
+      if (artworkId && !match?.params?.id) {
```

Comment:
Checking `!match?.params?.id` to ensure we don't set `source` again when we go to the details page from the artwork page (by pressing the backlink). In this scenario, submission already exists and updateConsignSubmission method will be called, we don't need to override the source there